api/python/tests/test_data_transfer.py

```diff
@@ -394,21 +394,20 @@ def test_multipart_upload(self):
             }
         )
 
-        with open(path, 'rb') as fd:
-            for part_num in range(1, 6):
-                self.s3_stubber.add_response(
-                    method='upload_part',
-                    service_response={
-                        'ETag': 'etag%d' % part_num
-                    },
-                    expected_params={
-                        'Bucket': 'example',
-                        'Key': 'large_file.npy',
-                        'UploadId': '123',
-                        'Body': fd.read(2048),
-                        'PartNumber': part_num
-                    }
-                )
+        for part_num in range(1, 6):
```

Comment:
out of curiosity, what's special about `6`?

Reply:
Just the file size divided by the chunk size.

(Could be written as an expression instead of hard-coding 6... but then again, it's good to verify it's a reasonable number - not 1 and not 6000.)